invenio_rdm_records/services/search_params.py

```diff
@@ -24,3 +24,33 @@ def apply(self, identity, search, params):
         if value is not None and value in [x.value for x in RecordDeletionStatusEnum]:
             search = search.filter("term", **{"deletion_status": value})
         return search
+
+
+class AdminParam(ParamInterpreter):
+    """Evaluates the 'status' parameter."""
+
+    def apply(self, identity, search, params):
+        """Evaluate the status parameter on the search."""
+        value = params.pop("is_admin", None)
```

Comment:
Is this correct?

Reply:
yes, there is no reason to pass the param down. It is also consistent with other query filters